go/client/prompts.go

```diff
@@ -42,4 +42,7 @@ const (
 	PromptDescriptorProvisionDeviceName
 	PromptDescriptorExportSecretKeyFromGPG
 	PromptDescriptorDeprovisionWhichUser
+	PromptDescriptorUpdateDo
+	PromptDescriptorUpdateAuto
+	PromptDescriptorUpdateSkipVersion
```

Comment:
Instead of skip I think we want snooze..

We never really want people to skip an update, it might be critical!?

For example, OSX nags you a lot but you can snooze instead of skipping.